### PR TITLE
OCPBUGS-37364: Detail unsupported routes for the External DNS Operator

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.12+

Issues:
* https://issues.redhat.com/browse/OCPBUGS-37635
* https://issues.redhat.com/browse/OCPBUGS-37364
* https://issues.redhat.com/browse/OCPBUGS-37636
* https://issues.redhat.com/browse/OCPBUGS-37443
* https://issues.redhat.com/browse/OCPBUGS-37440

Link to docs preview:
* [AWS Load Balancer Operator release notes](https://79724--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.html)
* [External DNS Operator release notes](https://79724--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/external_dns_operator/external-dns-operator-release-notes.html)
* [Creating DNS records on Azure](https://79724--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/external_dns_operator/nw-creating-dns-records-on-azure.html)
* [Creating DNS records on GCP](https://79724--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/external_dns_operator/nw-creating-dns-records-on-gcp.html)

- [x] SME has approved this change.
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: networking/external_dns_operator/nw-creating-dns-records-on-azure.adoc [hunk 1]
```diff
@@ -8,4 +8,9 @@ toc::[]
 
 You can create DNS records on Azure by using the External DNS Operator.
 
+[IMPORTANT]
+====
+Using the External DNS Operator on a {entra-first}-enabled cluster or a cluster that runs in {azure-full} Government (MAG) regions or {entra-first} is not supported.
```
Comment (lihongan) [inline networking/external_dns_operator/nw-creating-dns-records-on-azure.adoc:13]: looks duplicated `{entra-first}` ?  The latter one can be removed.
```
Using the External DNS Operator on a {entra-first}-enabled cluster or a cluster that runs in {azure-full} Government (MAG) regions is not supported
```

File: networking/external_dns_operator/nw-creating-dns-records-on-gcp.adoc [hunk 2]
```diff
@@ -6,6 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can create DNS records on GCP by using the External DNS Operator.
+You can create DNS records on {gcp-first} by using the External DNS Operator.
+
+[IMPORTANT]
+====
+Using the External DNS Operator on a cluster with {gcp-short}} Workload Identity enabled is not supported.
```
Comment (lihongan) [inline networking/external_dns_operator/nw-creating-dns-records-on-gcp.adoc:13]: extra `}` after `{gcp-short}`

File: networking/external_dns_operator/nw-creating-dns-records-on-azure.adoc (hunk 1 above)
Comment (dfitzmau) [reply networking/external_dns_operator/nw-creating-dns-records-on-azure.adoc:13]: :heavy_check_mark:

File: networking/external_dns_operator/nw-creating-dns-records-on-gcp.adoc (hunk 2 above)
Comment (dfitzmau) [reply networking/external_dns_operator/nw-creating-dns-records-on-gcp.adoc:13]: :heavy_check_mark:

**Comment by lihongan** [comment]
LGTM. thank you @dfitzmau

**Review by knobunc — APPROVED** [review]
/lgtm
/approve

File: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc [hunk 3]
```diff
@@ -9,6 +9,11 @@ toc::[]
 
 The AWS Load Balancer (ALB) Operator deploys and manages an instance of the `AWSLoadBalancerController` resource.
 
+[IMPORTANT]
+====
+The AWS Load Balancer (ALB) Operator is only supported on `x86_64` architecture.
```
Comment (jldohmann) [inline networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc:14]:
```suggestion
The AWS Load Balancer (ALB) Operator is only supported on the `x86_64` architecture.
```

File: networking/external_dns_operator/external-dns-operator-release-notes.adoc [hunk 4]
```diff
@@ -6,7 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The External DNS Operator deploys and manages `ExternalDNS` to provide name resolution for services and routes from the external DNS provider to {product-title}.
+The External DNS Operator deploys and manages `ExternalDNS` to provide name resolution for services and routes from the external DNS provider to {product-title}. 
+
+[IMPORTANT]
+====
+The External DNS Operator is only supported on `x86_64` architecture.
```
Comment (jldohmann) [inline networking/external_dns_operator/external-dns-operator-release-notes.adoc:13]:
```suggestion
The External DNS Operator is only supported on the `x86_64` architecture.
```

**Review by jldohmann — COMMENTED** [review]
LGTM! just a couple of nits 😄

File: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc (hunk 3 above)
Comment (dfitzmau) [reply networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc:14]: :heavy_check_mark:

File: networking/external_dns_operator/external-dns-operator-release-notes.adoc (hunk 4 above)
Comment (dfitzmau) [reply networking/external_dns_operator/external-dns-operator-release-notes.adoc:13]: :heavy_check_mark:

**Review by Srivaralakshmi — COMMENTED** [review]
lgtm

**Comment by Srivaralakshmi** [comment]
@dfitzmau Please share the manual CP PR links for the 4.12 and 4.13 branches. I will review and merge them.

**Pull request merged**